### ISSUE TITLE
Plugin::scaffold call is deprecated but README says it should be added to post-install commands

### ISSUE THREAD

**Issue by aaleksu** [drupal-composer/drupal-scaffold#113]
How is that possible that in 2.6.1 version in README you say that post-install-cmd should be adjusted with "DrupalComposer\DrupalScaffold\Plugn::scaffold()",
but Plugin::scaffold method has this as its first line: https://github.com/drupal-composer/drupal-scaffold/blob/2.6.1/src/Plugin.php#L96?

**Comment by greg-1-anderson** [comment]
Oh, if I had to guess I'd say that the README is out of date, but I don't really remember the fine details of this project, the entirety of which is deprecated.

Everyone should move to drupal/core-composer-scaffold. It is recommended, supported, and more performant.

**Comment by webflo** [comment]
@greg-1-anderson is right. You should look at ```drupal/core-composer-scaffold```. I skimmed through the README and could not find the part you mentioned about post-install-cmd.

**Comment by greg-1-anderson** [comment]
I updated the README to advise that this project is deprecated.

**Comment by marcvangend** [comment]
Thanks @greg-1-anderson. Could you elaborate a bit on the "everyone should move to drupal/core-composer-scaffold" part? What do I need to do to make the switch? Is is required to move to core-composer-scaffold when I upgrade my site to D9?

**Comment by greg-1-anderson** [comment]
Yes, drupal-composer/drupal-scaffold will not support Drupal 9 or Composer 2, so everyone will have to switch.
